specification/compute/resource-manager/readme.md

```diff
@@ -162,6 +162,32 @@ directive:
       - TrackedResourcePatchOperation
 
 ```
+### Tag: package-2018-06-exclude-gallery
+
+These settings apply only when `--tag=package-2018-06-exclude-gallery` is specified on the command line.
+
+``` yaml $(tag) == 'package-2018-06-exclude-gallery'
+input-file:
+- Microsoft.Compute/stable/2018-06-01/compute.json
+- Microsoft.Compute/stable/2018-06-01/runCommands.json
+- Microsoft.Compute/stable/2017-09-01/skus.json
+- Microsoft.Compute/stable/2018-04-01/disk.json
+- Microsoft.ContainerService/stable/2017-01-31/containerService.json

```

Comment:
@hyonholee - given [container service](https://github.com/Azure/azure-rest-api-specs/tree/master/specification/containerservices/resource-manager) is an RP by its own,  do you know the reason behind shipping specific version 2017-01-31 of container service along with every compute releases?